pkg/storage/gcs.go

```diff
@@ -48,28 +48,24 @@ func (options *GCSBackendOptions) apply(gcs *backup.GCS) error {
 }
 
 func defineGCSFlags(flags *pflag.FlagSet) {
-	flags.String(gcsEndpointOption, "", "Set the GCS endpoint URL")
+	// TODO: remove experimental tag if it's stable
+	flags.String(gcsEndpointOption, "", "(experimental) Set the GCS endpoint URL")
 	flags.String(gcsStorageClassOption, "",
-		`Specify the GCS storage class for objects.
+		`(experimental) Specify the GCS storage class for objects.
 If it is not set, objects uploaded are
 followed by the default storage class of the bucket.
 See https://cloud.google.com/storage/docs/storage-classes
 for valid values.`)
 	flags.String(gcsPredefinedACL, "",
-		`Specify the GCS predefined acl for objects.
+		`(experimental) Specify the GCS predefined acl for objects.
 If it is not set, objects uploaded are
 followed by the acl of bucket scope.
 See https://cloud.google.com/storage/docs/access-control/lists#predefined-acl
 for valid values.`)
 	flags.String(gcsCredentialsFile, "",
-		`Set the GCS credentials file path.
+		`(experimental) Set the GCS credentials file path.
 You can get one from
 https://console.cloud.google.com/apis/credentials.`)
-
-	_ = flags.MarkHidden(gcsEndpointOption)
-	_ = flags.MarkHidden(gcsStorageClassOption)
-	_ = flags.MarkHidden(gcsPredefinedACL)
-	_ = flags.MarkHidden(gcsCredentialsFile)
```

Comment:
The description of these options should better be shortened if they need to be visible.